internal/kubernetes/token/grpc.go

```diff
@@ -77,29 +77,26 @@ func authorizeConnection(ctx context.Context, kubeclient kubernetes.Clientset) e
 
 	md, ok := metadata.FromIncomingContext(ctx)
 	if !ok {
-		return status.Errorf(codes.Unauthenticated, "missing metadata")
+		return status.Error(codes.Unauthenticated, "missing metadata")
 	}
 
 	authHeader, ok := md["authorization"]
 	if !ok || len(authHeader) == 0 {
-		return status.Errorf(codes.Unauthenticated, "missing authorization token")
+		return status.Error(codes.Unauthenticated, "missing authorization token")
 	}
 
 	token := authHeader[0]
 	isValidated, err := validateBearerToken(ctx, token, kubeclient)
 	if !isValidated || (err != nil) {
-		return status.Errorf(codes.Unauthenticated, "invalid token")
+		return status.Error(codes.Unauthenticated, fmt.Sprint("invalid token: %w", err))
```

Comment:
this is where `status.Errorf()` can be used so that `fmt.Sprintf()` is not needed.

Reply:
Can you please address this as well?